common/src/main/java/com/microsoft/identity/common/internal/request/MsalBrokerRequestAdapter.java

```diff
@@ -232,6 +232,24 @@ public BrokerRequest brokerRequestFromSilentOperationParameters(@NonNull final S
         return requestBundle;
     }
 
+    public Bundle getRequestBundleForAllSsoTokens(
+            @NonNull final AcquirePrtSsoTokenCommandParameters parameters,
+            @NonNull final String negotiatedBrokerProtocolVersion) {
+
+        final Bundle bundle = new Bundle();
+        bundle.putString(AuthenticationConstants.Broker.NEGOTIATED_BP_VERSION_KEY, negotiatedBrokerProtocolVersion);
+        if (parameters.getRequestAuthority() != null) {
+            bundle.putString(AuthenticationConstants.Broker.REQUEST_AUTHORITY, parameters.getRequestAuthority());
+        }
+        if (parameters.getSsoUrl() != null) {
+            bundle.putString(AuthenticationConstants.Broker.BROKER_SSO_URL_KEY, parameters.getSsoUrl());
+        }
+        if (parameters.getCorrelationId() != null) {
+            bundle.putString(AuthenticationConstants.Broker.ACCOUNT_CORRELATIONID, parameters.getCorrelationId());
+        }
```

Comment:
addRequiredBrokerProtocolVersionToRequestBundle() is missing here.

Would caller provide required min version? if so, this call is needed and then is validated on broker side.